cli/cli/cli.go

```diff
@@ -452,6 +457,48 @@ func (c *CLI) logLevel() string {
 	return c.config.GetString("rexray.logLevel")
 }
 
+func (c *CLI) isConnSecured() {
+	if c.r != nil {
+		svr := c.r.API().ServerName()
+		fmt.Printf("*** %s ***\n", svr)
+	}
+}
+
+// handles the known_hosts error,
+// if error is ErrKnownHosts, stop execution to prevent unstable state
+func (c *CLI) handleKnownHostsError(err error) error {
```

Comment:
Hi @vladimirvivien,

Please ensure all code is written with an 80 character boundary and wrapped accordingly with emphasis on readability. Thank you.

In this case this function should invert almost all of the `if` blocks to return early instead of the deep indentations currently present.